### PR TITLE
Update Sites - Backup

### PR DESCRIPTION
**Pull request by ApCoder123**
- Added recovery documentation
- Specified custom-* values
- Updated a few doc links

### CODE REVIEW

**Comment by ApCoder123** [comment]
@Carlgo11 I think something might have gone wrong with overriding `redirect_uri_callback`?

**Comment by Carlgo11** [comment]
@ApCoder123 😩
I'll have a look at it when I get home in an hour.

**Comment by ApCoder123** [comment]
I've done some debugging and it looks like one of the redirects is to a relative path, which isn't handled by the new `redirect_uri_callback`. Adding the relevant code from [`default_redirect_uri_callback`](https://github.com/nahi/httpclient/blob/master/lib/httpclient.rb#L727) (which allows relative redirects) should solve the issue.

File: entries/m/mega.io.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "tfa": [
       "totp"
     ],
-    "documentation": "https://mega.io/blog_48",
+    "documentation": "https://mega.io/help/client/webclient/security-and-privacy#how-do-i-enable-two-factor-authentication-on-my-account-5bb525f9f1b70989348b45de",
```
Comment (phallobst) [inline entries/m/mega.io.json]:
```suggestion
    "documentation": "https://mega.io/help/client/webclient/security-and-privacy#5bb525f9f1b70989348b45de",
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/d/drive.google.com.json [hunk 2]
```diff
@@ -8,7 +8,12 @@
       "totp",
       "u2f"
     ],
+    "custom-software": [
+      "Google Prompts via Android",
+      "Google Prompts via Compatible iPhone App (Smart Lock, Gmail, Google app)"
```
Comment (phallobst) [inline entries/d/drive.google.com.json]: That's a bit of a mouthful, isn't it? Wouldn't `Google Prompts` or `Google push notification` be sufficient? Nobody will be updating the list of compatible apps anyway.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**